### PR TITLE
NS1: update ns1 for new txt handling

### PR DESCRIPTION
**Pull request by costasd**
* update auditrecords for succeeding tests
  * multiple segments not needed anymore. This was there to cover an edge case of a user setting segments in the dnsconfig file
  * strings can be longer than 255 octets, but not 0.
* update ns1 txt handling. No need for segmenting anymore, ns1 api handles this transparently.

### CODE REVIEW

File: providers/ns1/ns1Provider.go [hunk 1]
```diff
@@ -302,7 +302,7 @@ func buildRecord(recs models.Records, domain string, id string) *dns.Record {
 		if r.Type == "MX" {
 			rec.AddAnswer(&dns.Answer{Rdata: strings.Fields(fmt.Sprintf("%d %v", r.MxPreference, r.GetTargetField()))})
 		} else if r.Type == "TXT" {
-			rec.AddAnswer(&dns.Answer{Rdata: r.GetTargetTXTSegmented()})
+			rec.AddAnswer(&dns.Answer{Rdata: []string{r.GetTargetField()}})
```
Comment (tlimoncelli) [inline providers/ns1/ns1Provider.go:305]: I've been meaning to write up about this new function and when to use it.  For the meanwhile... trust me :-)

```suggestion
			rec.AddAnswer(&dns.Answer{Rdata: []string{r.GetTargetTXTJoined()}})
```

**Review by tlimoncelli — APPROVED** [review]

**Review by tlimoncelli — APPROVED** [review]

**Review by tlimoncelli — APPROVED** [review]

File: providers/ns1/ns1Provider.go (hunk 1 above)
Comment (costasd) [reply providers/ns1/ns1Provider.go]: sorry now coming to that - sure :)
